src/pipeline/postprocess/parseTQLRes.ts

```diff
@@ -159,28 +159,14 @@ export const parseTQLRes: PipelineOperation = async (req, res) => {
 
 		// todo: check if something weird happened
 		const expected = [...mutation.things, ...mutation.edges];
-		// console.log('expected', expected);
 		const result = expected
 			.map((exp) => {
 				//! reads all the insertions and gets the first match. This means each id must be unique
 				const currentNode = rawTqlRes.insertions?.find((y) => y.get(`${exp.$bzId}`))?.get(`${exp.$bzId}`);
-				// console.log('rawTqlRes.insertions', rawTqlRes.insertions);
-				// console.log('currentNode', currentNode);
 
 				// console.log('current:', JSON.stringify(x));
 
 				if (exp.$op === 'create' || exp.$op === 'update' || exp.$op === 'link') {
-					if (
-						!currentNode?.asThing().iid
-						// deletions are not confirmed in typeDB
-					) {
-						throw new Error(
```

Comment:
and this error solved me hours of debugging. Why should we skip it?

Reply:
When we set null and there is no other attribute update, there's no insertion. So rawTqlRes.insertion is undefined.

Reply:
This error was targetting deleted nodes instead of attrubutes. Lets merge it anyway we will do some deletion confirmation later and it might even get into typeDB roadmap soon anyway